201-representing-CNAB-in-OCI.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Representing CNAB bundles in OCI Registries
+weight: 201
+---
+
+# Representing CNAB bundles in OCI Registries
+
+This section describes how the bundle file and the information represented in the bundle MUST be represented when distributing bundles using OCI registries.
+
+A CNAB is made up of a descriptor file (`bundle.json`), one or more invocation images, and a list of zero or more component images. Each of these objects can be individually packaged into an [OCI manifest][oci-manifest] or [OCI index][oci-index]. An OCI index can then be used to combine these all into a single object.
+
+Runtimes and registries MAY choose to _relocate_ the images and invocation images referenced in the bundle to the registry where the bundle is pushed, but in its simplest form, a CNAB bundle MAY be represented in an OCI registry by the canonical `bundle.json` file descriptor, referenced in an OCI index (or manifest list).
```

Comment:
Should this be "to the repository where the bundle is pushed"? That's what cnab-to-oci does right?

Reply:
I would leave this discussion open, as both are valid and different use cases.